packages/material-ui/src/AvatarGroup/AvatarGroup.test.js

```diff
@@ -15,7 +15,8 @@ describe('<AvatarGroup />', () => {
     mount,
     muiName: 'MuiAvatarGroup',
     refInstanceof: window.HTMLDivElement,
-    skip: ['componentsProp'],
+    testVariantProps: { max: 5, spacing: 'medium', variant: 'circular' },
+    skip: ['componentsProp', 'themeComponents'],
```

Comment:
```suggestion
    skip: ['componentsProp'],
```

We should’t skip the theme components on this one

Reply:
@mnajdova In that case, the test case on theme variants is failing. How do we solve this?

1) <AvatarGroup />
       Material-UI component API
         theme: components
           respect theme's variants:

      AssertionError: expected <div class="MuiAvatarGroup-root css-usb1qg-MuiAvatarGroup-root" data-testid="without-props"></div> not to have computed style { 'margin-top': '13px' }Styles in JSDOM e.g. from `test:unit` are often misleading since JSDOM does not implement the Cascade nor actual CSS property value computation. If results differ between real browsers and JSDOM, skip the test in JSDOM e.g. `if (/jsdom/.test(window.navigator.userAgent)) this.skip();`
Browsers can compute shorthand properties differently. Prefer longhand properties e.g. `borderTopColor`, `borderRightColor` etc. instead of `border` or `border-color`.

Reply:
Ok, it took me some time to figure out what was wrong here 😄 You have used in the `testVariantProps` only default values, and we have tests what will happen if the props are provided and not, but because they are always provided (being defaults) the test was failing. Please change any of the values you have in the `testVariantProps`. For me this worked:

```diff
diff --git a/packages/material-ui/src/AvatarGroup/AvatarGroup.test.js b/packages/material-ui/src/AvatarGroup/AvatarGroup.test.js
index 0347e5a1e2..8a33526715 100644
--- a/packages/material-ui/src/AvatarGroup/AvatarGroup.test.js
+++ b/packages/material-ui/src/AvatarGroup/AvatarGroup.test.js
@@ -15,8 +15,8 @@ describe('<AvatarGroup />', () => {
     mount,
     muiName: 'MuiAvatarGroup',
     refInstanceof: window.HTMLDivElement,
-    testVariantProps: { max: 5, spacing: 'medium', variant: 'circular' },
-    skip: ['componentsProp', 'themeComponents'],
+    testVariantProps: { max: 10, spacing: 'medium', variant: 'circular' },
+    skip: ['componentsProp'],
   }));

   it('should display all the avatars', () => {

```